app.js

```diff
@@ -197,7 +198,11 @@ app.get('/auth/github', passport.authenticate('github'));
 app.get('/auth/github/callback', passport.authenticate('github', { failureRedirect: '/login' }), (req, res) => {
   res.redirect(req.session.returnTo || '/');
 });
-app.get('/auth/google', passport.authenticate('google', { scope: 'profile email' }));
+app.get('/auth/google', passport.authenticate('google', { scope: [
+  'profile',
+  'email',
+  'https://www.googleapis.com/auth/drive'
+  ], accessType: 'offline', prompt: 'consent'}));
```

Comment:
I think chaging these lines to a single line `app.get('/auth/google', passport.authenticate('google', { scope: ['profile', 'email', 'https://www.googleapis.com/auth/drive'], accessType: 'offline', prompt: 'consent' }));` is probably going to fix the build error.  You can locally run lint to check by:  `npm run lint`